server/src/test/scala/com/cloudera/livy/server/recovery/ZooKeeperStateStoreSpec.scala

```diff
@@ -96,6 +96,21 @@ class ZooKeeperStateStoreSpec extends FunSpec with LivyBaseUnitTestSuite {
       }
     }
 
+    it("get should retrieve retry policy configs") {
+      withMock { f =>
+        mockExistsBuilder(f.curatorClient, true)
+
+        val getDataBuilder = mock[GetDataBuilder]
+
+        assert(f.stateStore.retryPolicy != null)
+        assert(f.stateStore.retryPolicy.getN === 5) // default value is 5 retries with 100ms sleeps
```

Comment:
Mind if you change the retry value in `conf` to a value different from the default at the top of the test, then verify against it here?